### PR TITLE
[Bulk] Switch to Stripe Checkout.

### PR DESCRIPTION
**Pull request by dracos**
As checkout lets you specify tax. Test Stripe has a bulk Price set up.

### CODE REVIEW

File: conf/general.yml-example [hunk 1]
```diff
@@ -54,6 +54,9 @@ STRIPE_PUBLIC_KEY: ''
 STRIPE_API_VERSION: ''
 STRIPE_TAX_RATE: ''
 
+BULK_LOOKUP_AMOUNT: 50
+BULK_LOOKUK_PRICE_ID: 'price_123'
```
Comment (davea) [inline conf/general.yml-example]: Small typo:
```suggestion
BULK_LOOKUP_PRICE_ID: 'price_123'
```

File: bulk_lookup/views.py [hunk 2]
```diff
@@ -107,45 +99,60 @@ def get_context_data(self, form, **kwargs):
...
+    context = {}
+    return_url = ''.join([
+        'https://',
+        get_current_site(None).domain,
+        f'/bulk/{bulk_lookup.id}/{{CHECKOUT_SESSION_ID}}',
+    ])
+    description = bulk_lookup.description or 'Bulk lookup'
+    session = stripe.checkout.Session.create(
+        line_items=[{
+            'price': 'price_1Q7y4zLoAAr9vgdbwPHVezZ6',
```
Comment (davea) [inline bulk_lookup/views.py]: Should this come from `settings.BULK_LOOKUP_PRICE_ID` rather than being hardcoded?

**Review by davea — CHANGES_REQUESTED** [review]

File: bulk_lookup/views.py (hunk 2 above)
Comment (dracos) [reply bulk_lookup/views.py]: Would be good if I used the new config variable somewhere, wouldn't it, doh!

**Review by davea — APPROVED** [review]

File: bulk_lookup/views.py [hunk 3]
```diff
@@ -107,45 +99,60 @@ def get_context_data(self, form, **kwargs):
             raise WizardError
         context['num_good_rows'] = pc_data['num_rows'] - pc_data['bad_rows']
         if self.steps.current == 'personal_details':
-            context['price'] = settings.BULK_LOOKUP_PRICE
+            context['price'] = settings.BULK_LOOKUP_AMOUNT
             if self.object and self.object.plan.id == settings.PRICING[-1]['plan']:
```
Comment (dracos) [inline bulk_lookup/views.py:103]: As the looking up of self.object above in get_form_kwargs was removed, this was never true - added a similar is_authenticated/lookup of self.object here afterwards.

**Pull request merged**